testdata/dnn/darknet/generate_darknet_models.py

```diff
@@ -37,4 +35,4 @@ def genTestData(name, inpShape, outShape):
 genTestData('route_multi', [1, 6, 3, 6], [1, 4, 2, 3])
 genTestData('relu', [1, 2, 3, 4], [1, 2, 3, 4])
 genTestData('sam', [1, 2, 3, 4], [1, 4, 3, 4])
-
+genTestData('crop', [1, 2, 3, 4], [1, 2, 3, 4])
```

Comment:
Should shapes match the values from the .cfg file?

Reply:
> Should shapes match the values from the .cfg file?

Yes. the `inpShape` should match the values from `.cfg file`. Also can you suggest me what values for `outShape` can I pass as the parameter.

Reply:
I think output shape should match `crop_height` and `crop_width`. So `genTestData('crop', [1, 3, 5, 6], [1, 3, 3, 3])`